src/app/details/details.component.html

```diff
@@ -3,7 +3,7 @@
   class="details__current-weather"
   [weatherInfo]="weatherInfo"
 ></app-current-weather>
-<app-btn-groups class="details__details-btns"></app-btn-groups>
+<app-btn-groups class="details__details-btns" [weatherInfo]="weatherInfo"></app-btn-groups>
```

Comment:
we don't need to send data in btn-groups

Reply:
but we need to create links for forecasts, if we won't pass weatherInfo with city, there's no way we can determine for which city we should render forecast